hmy/api_backend.go

```diff
@@ -362,3 +362,12 @@ func (b *APIBackend) GetValidatorSelfDelegation(addr common.Address) *big.Int {
 	}
 	return wrapper.Delegations[0].Amount
 }
+
+// GetShardState ...
+func (b *APIBackend) GetShardState() *shard.State {
```

Comment:
`shard.State` is a slice (defined as `[]shard.Committee`), so please just return the slice itself, not a pointer to it.  Also please return an error code:
```suggestion
func (b *APIBackend) GetShardState() (shard.State, error) {
```